raiden/message_handler.py

```diff
@@ -31,27 +31,27 @@
 from raiden.transfer.state import balanceproof_from_envelope
 from raiden.transfer.state_change import ReceiveDelivered, ReceiveProcessed, ReceiveUnlock, ReceiveUnlockLight
 from raiden.utils import pex, random_secret
-from raiden.utils.typing import MYPY_ANNOTATION
+from raiden.utils.typing import MYPY_ANNOTATION, Address
 
 log = structlog.get_logger(__name__)  # pylint: disable=invalid-name
 
 
 class MessageHandler:
-    def on_message(self, raiden: RaidenService, message: Message, is_light_client: bool = False) -> None:
+    def on_message(self, raiden: RaidenService, message: Message, node_address: Address, is_light_client: bool = False) -> None:
```

Comment:
`node_address` seems ambiguous. is this the sender or the recipient of the message?